modules/EsminiAdapter/src/esminiadapter.cpp

```diff
@@ -20,6 +20,8 @@
 #include "rclcpp/wait_for_message.hpp"
 #include "trajectory.hpp"
 #include "string_utility.hpp"
+#include "util.h"
```

Comment:
Possible to
```suggestion
```
?

Reply:
needed for UtilGetOscDirectoryPath, this can be addressed in a future pr where we break out util into e.g. fileutils etc.